res/controllers/Reloop-Beatmix-2-4-scripts.js

```diff
@@ -0,0 +1,773 @@
+/************************  GPL v2 licence  *****************************
+ * Reloop Beatmix controller script
+ * Author: Sébastien Blaisot <sebastien@blaisot.org>
+ *
+ **********************************************************************
+ * User References
+ * ---------------
+ * Wiki/manual : http://www.mixxx.org/wiki/doku.php/reloop_beatmix_4
+ * support forum : http://mixxx.org/forums/viewtopic.php?f=7&t=8428
+ *
+ * Revision history
+ * ----------------
+ * 2016-07-31 - Initial revision for Mixxx 2.1.0
+ ***********************************************************************
+ *                           GPL v2 licence
+ *                           --------------
+ * Reloop Beatmix controller script script 1.0 for Mixxx 2.1.0
+ * Copyright (C) 2015 Sébastien Blaisot
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ***********************************************************************/
+////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                               //
+////////////////////////////////////////////////////////////////////////
+/* global engine                                                      */
+/* global script                                                      */
+/* global print                                                       */
+/* global midi                                                        */
+////////////////////////////////////////////////////////////////////////
+// Global variables and declarations.
+// ========================================================
+function ReloopBeatmix24() {}
+
+ReloopBeatmix24(); // Very important ! Initializes the declared function, yep
```

Comment:
IMO it's clearer to write:
``var ReloopBeatmix24 = {};``
There is no need for it to be a function, it just needs to be a declared variable that is an object. (See http://mixxx.org/forums/viewtopic.php?p=29447#p29447 for a little more detail)